### ISSUE TITLE
Feature Request: Show effective caller id name on the extensions page

### ISSUE THREAD

**Issue by jlcrews** [fusionpbx/fusionpbx#2051]
Please add the extension effective caller id name on the page you see when you click Accounts, Extensions.

There are lots of name and number fields on the extension and it could be handy to show many of those...but it may be too much.

If there was a way to toggle which fields show up on the extensions page (and perhaps other summary pages for that matter that would be super nice...but may be a lot of work for perhaps few people asking for this).

Hoping this is quick and easy request.

Thanks!

**Comment by markjcrane** [comment]
We need a tool that allows selecting which columns we want to display on pages that have a lot of columns. In this way we can accommodate more in the future. So far we have used permissions to show or hide columns not certain it is the best way. Probably want to add a button on the page 'Columns' which shows a tool with check boxes to choose the columns to show. Or something like this.

**Comment by moteus** [comment]
Agree. E.g. for me it will be good to display number-alias instead of extension number and also order extesons by this number.
